docs/development/near-cli.md

```diff
@@ -71,8 +71,8 @@ _Click on a command for more information and examples._
 
 #### Mac and Linux
 
-1. Install `npm` [[ click here ]](https://www.npmjs.com/get-npm)
-2. Install `NodeJS` [[ click here ]](https://nodejs.org/en/download)
+1. Install `npm` It's recommended to use a package manager like `nvm` as sometimes there are issues using Ledger due to how OS X handles node packages related to USB devices. [[ click here ]](https://nodejs.org/en/download/package-manager/)
```

Comment:
did you mean to remove the `npm` installation link?  Was your intention to have the node package manager info for step 2?

Reply:
nvm will install node and npm, then I wanted to make sure they install node 12

Reply:
ahh gotcha!